src/compiler/AST/Canonical.elm

```diff
@@ -47,11 +52,3 @@ var qualifier name =
         { qualifier = qualifier
         , name = name
         }
-
-
-lambda : VarName -> Expr -> Expr
-lambda argument body =
```

Comment:
Was there a good reason for removing this function? (Its only usage was Desugar.curryLambda I believe, but still)
There totally might have been and I just missed it, dunno...

Reply:
@Warry Thoughts?